frontend/frames/buttons.py

```diff
@@ -0,0 +1,36 @@
+from tkinter import Button, font, Frame, PhotoImage
+
+class ButtonFrame(Frame):
+    '''Frame with buttons'''
+    def __init__(self, window, base, **kwargs):
+        Frame.__init__(self, window, **kwargs)
```

Comment:
That one should looks like
```
super().__init__(self, window, **kwargs)
```
see: https://docs.python.org/3/library/functions.html#super

Reply:
That works well without `self` argument i.e:
```
super().__init__(window, **kwargs)
```